File: spring boot/src/main/java/com/krishna/Stegano/payloads/DecodeResponse.java
package com.krishna.Stegano.payloads;

import lombok.Data;


@Data
public class DecodeResponse {
    private String originalString;
    private byte[] fileContent;

    public DecodeResponse(String originalString, byte[] fileContent) {
        this.originalString = originalString;
        this.fileContent = fileContent;
    }

}

File: client/lib/model/decode_response.dart
import 'dart:convert';
import 'dart:typed_data';

class DecodeResponse {
  final String data;
  final Uint8List byteData;

  DecodeResponse(this.data, this.byteData);

  factory DecodeResponse.fromJson(Map<String, dynamic> json) {
    String bytes= json["fileContent"];
    Uint8List imageData = base64Decode(bytes);
    return DecodeResponse(json['originalString'], imageData);
  }
}
